### ISSUE TITLE
Custom Event Sim

### ISSUE THREAD

**Issue by tervay** [avgupta456/statbotics#208]
Hi, thanks so much for all the work on this. I was wondering if it would be possible to build out a webpage to simulate event rankings given a team list and a year (without a schedule). This feature would keep me entertained for a long time.

**Comment by avgupta456** [comment]
Yeah that sounds like a nice feature, I'll try and work on it soon. Hopefully before champs :)

**Comment by avgupta456** [comment]
Just built it, sorry for the delay!

https://www.statbotics.io/hypothetical#event
